docs_app/content/tutorial/6-chaining-operators.md

```diff
@@ -0,0 +1,14 @@
+# 6. Chaining Operators
+
+## TLDR:
+
+- `pipe` is a functional programming concept.
+- Use `Observable` [`pipe`](API).
+- You can create many small chains to shorten larger chains.
+
+## Chaining Operators
+
+If an operator, as state above, is a "thing that can perform am operation", on a collection, then we might be able to go back to our [collection of apples](#Collections). Let's had a machine that would sort the apples (bad and good), and we also had a machine that could peel apples, and perhaps a machine that could slice the apples. We could then, in theory, chain them all together with some sort of conveyor and sort, peel, and slice the apples all in one step.
+
+
+WIP: MOre to do here.
```

Comment:
I'm not sure what else is on your list to cover, but here are a few thoughts:

- Subject/BehaviorSubject (Later in Essentials). Hot vs cold? uni-cast vs multi-cast? Lazy vs ?
- Error handling within a pipe. (Essentials)
- Encapsulating error handling to allow an outer Observable to continue. Example, user selects a product from a list. The action emits the productId into a stream which issues a get request to a back end. The back end generates an error, but outer Observable (handling product selection) should continue to work. (Advanced)
- Retry (Probably Advanced)
- Style guide? (naming guidelines, piping guidelines, using Subject and asObservable, etc)
- Best practices? (No nested subscribes, limit on number of operators in a single pipe?, Mike's quote: "Reactive programming stops at subscribe"?, state management with RxJS?, caching/refreshing a cache?, etc)
- Testing?